### PR TITLE
UPDATE - Translation to the Brazilian Portuguese language 

### PR DESCRIPTION
**Pull request by laetemn**
New update of the translation to the Brazilian Portuguese language.

### CODE REVIEW

**Comment by bjorn** [comment]
They, thanks for the update! I think I forgot to notify you about the string freeze, in which case I'm really sorry! I had to announce while on holiday and didn't have my administration with me, and then I forgot to check later.

I will merge this into the `0.15` branch and include it in a 0.15.2 update.

If you want, you could try to get rid of the merge commits included in this PR and reopen it as a request to merge into 0.15, but this is all optional, I can cherry-pick it into the right branch without the merge commits no problem!

**Comment by laetemn** [comment]
Hummm.....

I just installed the 64-bit version of Tile in Windows 10 to test translation (Brazilian Portuguese), and I realized that the software name appears incorrectly in the Windows 10 menu.

tiled-error

**Comment by bjorn** [comment]
> I have found no reference (text) in the 'tiled_pt.ts' file to the following texts:
> 
>   * Texts of the Update tab located in the Preferences window;
>   * Text "Object Types Editor" located in the View menu;
>   * Texts of the window "Object Types Editor".

That is expected, because these are strings that were added after Tiled 0.15 was released. Hence, I also said that this translation update should be merged to the `0.15` branch, and I will release it with Tiled 0.15.2, which does not include the above texts yet.

> I just installed the 64-bit version of Tiled in Windows 10 to test translation (Brazilian Portuguese), and I realized that the software name appears incorrectly in the Windows 10 menu.

Right, I had seen this myself as well but had forgotten, so thanks for the reminder! This is a small problem in the new Windows installer (not used yet for Tiled 0.15.x), I'll try a fix shortly.

File: translations/tiled_pt.ts [hunk 1]
```diff
@@ -2488,12 +2488,12 @@ Linha %1, coluna %2</translation>
...
         <location line="+250"/>
         <source>Tileset Columns Changed</source>
-        <translation type="unfinished"></translation>
+        <translation>Colunas do Tileset Alteradas</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>The number of tile columns in the tileset &apos;%1&apos; appears to have changed from %2 to %3. Do you want to adjust tile references?</source>
-        <translation type="unfinished"></translation>
+        <translation>O número de colunas do tile no tileset &apos;%1&apos; parece ter mudado de%2 para %3. Você deseja ajustar as referências do tile?</translation>
```
Comment (bjorn) [inline translations/tiled_pt.ts]: I assume there should be a space before the `%2` here, which I'll add before pushing the change.

**Comment by bjorn** [comment]
Translation update pushed in commit d1d2727f49d5d64b59efa1e56c857ca65197d964, thanks!

**Comment by bjorn** [comment]
> This is a small problem in the new Windows installer (not used yet for Tiled 0.15.x), I'll try a fix shortly.

Hrm, I tried to fix this, but whatever other text I tried to put in there caused the whole start menu shortcut to not be there at all. Will need to get some sleep now and look into this again later...

**Pull request closed without merging**